pennylane/devices/default_gaussian.py

```diff
@@ -695,6 +693,19 @@ def __init__(self, wires, *, shots=1000, hbar=2, analytic=True):
 
         self.reset()
 
+    @classmethod
+    def capabilities(cls):
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            model="cv",
+            supports_analytic_computation=True,
+            supports_finite_shots=True,
+            returns_probs=False,
+            returns_state=False,
```

Comment:
Perhaps we could set `returns_state=False` in the base `Device`, and then only turn it on when needed.

Reply:
The reason why I reduced the default values in the base device is that not having the value is better than misreporting. Also, it would set this value for all devices, and it would be better to go through them one by one...